src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2413,6 +2412,8 @@ static bool updateContextAttributeItem
   bool&                     entityModified,
   std::string*              currentLocAttrName,
   BSONObjBuilder*           geoJson,
+  mongo::Date_t*            dateExpiration,
+  bool*                     replaceDate,
```

Comment:
Maybe better: replaceExpirationDate

(There could be other date attributes, now or in the future, and replaceDate would be ambiguous)